doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -37,8 +38,8 @@ See the following table for more details:
    :widths: auto
 
    * -
-     - Supported
-     - Experimental
+     - Production
+     - Development (Experimental)
```

Comment:
```suggestion
     - Development
```
We don't mention "--" next to "Not supported" all the time, so let's use the proper term without its alternative.